### PR TITLE
try to fix selfLink issues

### PR DESCRIPTION
**Pull request by qcfe**
# Description
Possible emergency fix to be tested for the dashboard which has broken after the update to Kubernetes 1.21

### CODE REVIEW

**Comment by qcfe** [comment]
/undeploy-staging
broken

File: dashboard/views/crownLabs/CrownLabsHome.js [hunk 1]
```diff
@@ -11,13 +11,41 @@ import TemplateForm from './TemplateForm';
...
+  const setInstancesProfessor = (instances) => {
+    instances.forEach((i) => {
+      t.metadata.selfLink = `/crownlabs.polito.it/v1alpha2/namespaces/${i.metadata.namespace}/instances/${i.metadata.name}`;
+    });
+    setInstancesProfessorRaw(instances);
+  };
+
+  const setTenants = (tenants) => {
+    tenants.forEach((t) => {
+      t.metadata.selfLink = `/crownlabs.polito.it/v1alpha2/namespaces/${t.metadata.namespace}/templates/${t.metadata.name}`;
```
Comment (GabriFila) [inline dashboard/views/crownLabs/CrownLabsHome.js:44]:
```suggestion
      t.metadata.selfLink = `/crownlabs.polito.it/v1alpha1/tenants/${t.metadata.name}`;
```

**Pull request closed without merging**